scapy/layers/tls/automaton.py

```diff
@@ -7,7 +7,7 @@
 The _TLSAutomaton class provides methods common to both TLS client and server.
 """
 
-from __future__ import print_function
+import logging
```

Comment:
Have you seen https://github.com/secdev/scapy/blob/master/scapy/error.py
Scapy has its own logging handlers. You should use them